@airswap/metadata: error handling

dmosites opened the pull request:
Return a `message` and `url` for errors; remove AST check in tests

hmalik88 reviewed (commented):
LGTM, however one of the tests is failing: 

<img width="805" alt="Screen Shot 2021-09-04 at 11 54 37 AM" src="https://user-images.githubusercontent.com/41640681/132100711-b79ab71e-d6e9-45db-8dea-cd4a5321cbb9.png">

I had a sandbox from before that I was using to work on the metadata package and I saw this in the console when running your code: 

<img width="474" alt="Screen Shot 2021-09-04 at 11 54 51 AM" src="https://user-images.githubusercontent.com/41640681/132100776-936b92a2-c2ec-4405-b851-04d4b2234fac.png">

Weird thing is that those URLs work fine in my browser, anyway those two are the lists are the ones that have WETH, so that explains the test failing.

dmosites commented:
Those servers must look at the request headers to decide whether to respond OK. Given that they've started erroring out in test environments I've removed the impacted mainnet tests and left the rinkeby ones so that we still test those functions. Also removed the shim. 👍
